addons/rose/addon/styles/hds/overrides.scss

```diff
@@ -31,6 +31,13 @@
     color: var(--token-form-indicator-optional-color);
   }
 
+  // Override for sortable HDS table columns that ignore selected theme in app
+  // and use the system default theme causing color mismatch if selected theme
+  // is not the system default theme
+  .hds-table__th-sort button {
```

Comment:
None blocking comment: Did we ask/mention this to design system team (HDS) ?

Reply:
Not yet. We are a bit behind on our HDS version so we need to update first before bringing this to the team.

Reply:
@cameronperera good call 😉